### DISCUSSION TITLE
list only work for Lua files

### DISCUSSION THREAD

**Discussion by RazaGR** [LazyVim/LazyVim#1560] [Q&A] [answered]
in `.go` files I get a `>` on space but not in Lua files, I had to add in options `vim.opt.list = false`:

`lua file`
<img width="611" alt="Screenshot 2023-10-02 at 22 51 38" src="https://github.com/LazyVim/LazyVim/assets/30994076/ed27e061-ae2e-43aa-a9dc-0730dd4d1a2b">

`go file`
<img width="191" alt="Screenshot 2023-10-02 at 22 51 55" src="https://github.com/LazyVim/LazyVim/assets/30994076/8bcb07fa-577e-44e1-a2de-3ef81cd1b3db">

**Comment by folke** [comment, marked as answer]
Was aready fixed, but I didn't push a new release yet. Just did https://github.com/LazyVim/LazyVim/pull/1553

**Reply by RazaGR** [reply]
yea, updated and seems fixed...Thanks
